extensions/CheckUser/CheckUser.hooks.php

```diff
@@ -126,7 +126,7 @@ public static function updateCUEmailData( $to, $from, $subject, $text ) {
 			'cuc_title'      => '',
 			'cuc_minor'      => 0,
 			'cuc_user'       => $userFrom->getId(),
-			'cuc_user_text'  => $userFrom->getName(),
+			// 'cuc_user_text'  => $userFrom->getName(), // SUS-3080 - this column is redundant
 			'cuc_actiontext' => wfMsgForContent( 'checkuser-email-action', $hash ),
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this legacy wfMsg* function call with one of the new i18n functions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)

[project-id]: wikia:app